internal/pkg/testing/helpers.go

```diff
@@ -18,13 +18,19 @@ import (
 	"testing"
 
 	"github.com/GoogleContainerTools/kpt/internal/pkg"
+	"github.com/GoogleContainerTools/kpt/internal/util/pathutil"
 	"github.com/stretchr/testify/assert"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
 )
 
 // CreatePkgOrFail creates a new package from the provided path. Unlike the
 // pkg.New function, it fails the test instead of returning an error.
 func CreatePkgOrFail(t *testing.T, path string) *pkg.Pkg {
-	p, err := pkg.New(path)
+	absPath, _, err := pathutil.ResolveAbsAndRelPaths(path)
```

Comment:
`fsys` should be provided to `CreatePkgOrFail` so that the test called can provide fsys.